index.js

```diff
@@ -98,6 +101,13 @@ module.exports = function (config) {
       // and if bddStep after event is emitted, it should have passed
       if (report.step.result.status === 'skipped') report.step.result.status = 'passed';
     });
+    recorder.add('Set step end time and duration', async () => {
+      if (report && report.step) {
+        report.step.end_time = Date.now();
+        // calculate the duration in nanoseconds for cucumber-html-reporter
+        report.step.result.duration = (report.step.end_time - report.step.start_time) * 1000000;
```

Comment:
can we make this time multiplier a config value
I know some reporters like cucumber-html-reporter use nano seconds, however others use milliseconds

Reply:
Done